src/py/flwr/client/supernode/app.py

```diff
@@ -309,6 +321,23 @@ def _parse_args_run_client_app() -> argparse.ArgumentParser:
     return parser
 
 
+def _parse_args_flwr_clientapp() -> argparse.ArgumentParser:
```

Comment:
Let's use this directly in the `flwr_clientapp` function above and log the arguments that were passed

Reply:
I've set the log level to `DEBUG`. Otherwise the outputs will be quite verbose for every round of `ClientApp` communication.